homeassistant/components/allpowers_ble/__init__.py

```diff
@@ -0,0 +1,95 @@
+"""The Allpowers BLE integration."""
+
+import logging
+
+from allpowers_ble import AllpowersBLE
+from bleak.exc import BleakError
+from bleak_retry_connector import get_device
+
+from homeassistant.components import bluetooth
+from homeassistant.components.bluetooth.match import ADDRESS, BluetoothCallbackMatcher
+from homeassistant.config_entries import ConfigEntry
+from homeassistant.const import CONF_ADDRESS, EVENT_HOMEASSISTANT_STOP, Platform
+from homeassistant.core import Event, HomeAssistant, callback
+from homeassistant.exceptions import ConfigEntryNotReady
+
+from .const import DOMAIN
+from .coordinator import AllpowersBLECoordinator
+from .models import AllpowersBLEData
+
+PLATFORMS: list[Platform] = [Platform.SWITCH, Platform.SENSOR]
```

Comment:
Please reduce this PR to a single platform.

See also: 

- <https://developers.home-assistant.io/docs/review-process>
- https://developers.home-assistant.io/docs/creating_component_code_review#5-make-your-pull-request-as-small-as-possible

Additional platforms can be added in follow-up PRs